projects/core/styles/components/link.less

```diff
@@ -59,13 +58,24 @@
         background: color-mix(in lch, currentColor 12%, transparent);
     });
 
-    @media @tui-mouse {
-        &[data-appearance='']:hover {
-            opacity: 0.7;
-        }
+    .appearance-hover({
+        text-decoration: underline;
+    });
+
+    [tuiTheme='dark'] &[data-appearance=''] {
+        text-decoration-color: var(--tui-service-underline-grayscale, rgba(255, 255, 255, 0.32));
+        color: var(--tui-text-primary);
+    }
+
+    &[data-appearance=''],
+    [tuiTheme='light'] &[data-appearance=''] {
+        text-decoration: underline;
+        text-decoration-color: var(--tui-service-underline-grayscale, rgba(0, 0, 0, 0.32));
     }
 
-    &[data-appearance='']:active {
-        opacity: 0.7;
+    &[data-appearance='action'] {
+        .appearance-hover({
```

Comment:
Let's not do this, might be just a mistake in the specs, I notified designers.

Reply:
fixed